pkg/cmd/server/crypto/crypto.go

```diff
@@ -455,7 +453,20 @@ func NewKeyPair() (crypto.PublicKey, crypto.PrivateKey, error) {
 }
 
 // Can be used for CA or intermediate signing certs
-func newSigningCertificateTemplate(subject pkix.Name) (*x509.Certificate, error) {
+func newSigningCertificateTemplate(subject pkix.Name, expireDays int) (*x509.Certificate, error) {
+
+	var caLifetimeInDays = defaultCACertificateLifetimeInDays
+	if expireDays > 0 {
+		caLifetimeInDays = expireDays
+	}
+
+	if caLifetimeInDays >= maxRecommendedLifetimeInDays {
```

Comment:
Ok, I agree, but for some strange reason I heard about 10 years and the same period specified in the Trello card..